test/src/tests.js

```diff
@@ -948,8 +948,55 @@ describe('Adwords forwarder', function () {
                 done();
             });
 
-            it('should set enhanced conversion data on commerce events', function(done) {
-                var successMessage = mParticle.forwarder.process({
+            it('should set enhanced conversion data as strings on custom events', function(done) {
+                mParticle.forwarder.process({
+                    EventName: 'Homepage',
+                    EventDataType: MessageType.PageEvent,
+                    EventCategory: EventType.Navigation,
+                    CustomFlags: {
+                        'GoogleAds.ECData': JSON.stringify({
+                            email: 'test@gmail.com',
+                            phone_number: '1-911-867-5309',
+                            first_name: 'John',
+                            last_name: 'Doe',
+                            home_address: {
+                                street: '123 Main St',
+                                city: 'San Francisco',
+                                region: 'CA',
+                                postal_code: '12345',
+                                country: 'US',
+                            },
```

Comment:
Since this is used repeatedly in the tests, let's set it as a constant at the top of the file and reference it.